x/fundraising/keeper/bid.go

```diff
@@ -27,6 +27,10 @@ func (k Keeper) PlaceBid(ctx sdk.Context, msg *types.MsgPlaceBid) (types.Bid, er
 		return types.Bid{}, types.ErrInvalidAuctionStatus
 	}
 
+	if msg.Price.LT(auction.GetMinBidPrice()) {
```

Comment:
This PR currently has two changes: 1) min bid amount and 2) min bid price. Is 2) min bid price also a target to change?